editors/code/src/extension.ts

```diff
@@ -89,11 +92,11 @@ export function activate(context: vscode.ExtensionContext) {
     const allNotifications: Iterable<
         [string, lc.GenericNotificationHandler]
     > = [
-        [
-            'rust-analyzer/publishDecorations',
-            notifications.publishDecorations.handle
-        ]
-    ];
+            [
```

Comment:
Drive-by formatting applied when I formatted the entire document. Is that okay or should I revert it?

Reply:
I think the `.ts` formatting is nailed down by prettier in the CI

Reply:
Did `yarn prettier --write`, this should fix any remaining formatting issues